server/models/Profile.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    lastName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    description: String,
+    gender: {
+        type: String,
+        required: true,
+        enum: ['Male', 'Female']
+    },
+    birthDate: Date,
+    email: {
```

Comment:
The User model already has an 'email' field. We wouldn't want to duplicate fields across models as it may cause some confusion in the future :)